app/assets/javascripts/discourse/controllers/preferences/users.js.es6

```diff
@@ -1,10 +1,44 @@
 import PreferencesTabController from "discourse/mixins/preferences-tab-controller";
 import { popupAjaxError } from "discourse/lib/ajax-error";
+import showModal from "discourse/lib/show-modal";
+import User from "discourse/models/user";
 
 export default Ember.Controller.extend(PreferencesTabController, {
   saveAttrNames: ["muted_usernames", "ignored_usernames"],
-
+  ignoredUsernames: Ember.computed.alias("model.ignored_usernames"),
   actions: {
+    ignoredUsernamesChanged(previous, current) {
```

Comment:
If possible I would change naming here. It feels like an event, when it's actually an action. This name doesn’t tell me what it's doing.

Reply:
Good point 👍